clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoOptions.java

```diff
@@ -64,12 +66,13 @@ public Options options() {
     options.addOption(createArgOption("V", VALUE, "property value"));
     options.addOption(
         createArgOption(
-            "g", PROVIDER, "provider one of hadoop, hive, mysql, postgres, iceberg, kafka"));
+            "t", PROVIDER, "provider one of hadoop, hive, mysql, postgres, iceberg, kafka"));
+    options.addOption(createArgOption("l", USER, "user name"));
+    options.addOption(createArgOption("g", GROUP, "group name"));
 
     // Properties option can have multiple values
     Option properties =
-        createArgOption("p", PROPERTIES, "comma separated property name/value pairs");
-    properties.hasArgs();
+        Option.builder("p").longOpt(PROPERTIES).desc("property name/value pairs").hasArgs().build();
```

Comment:
"comma separated" is a necessary description for the properties I think, why not keep it?

Reply:
The option has been updated, and they no longer have to be comma separated - see the description in cli.md.

Reply:
I see; This change is irrelavant with current PR, now mixed with other changes; Next time please use a seperate PR, that will be helpful for review as well as code management (revert, cherry-pick, etc).